### DISCUSSION TITLE
Jupyter Login Help

### DISCUSSION THREAD

**Discussion by roshandk** [cooperrc/engineering-dynamics#6] [Q&A]
Hey all, for some reason when I try to log on to Jupyter it keeps giving me the 403 Forbidden page. I've changed browsers and even tried on my phone and still doesn't work. 
![image](https://github.com/cooperrc/engineering-dynamics/assets/157177646/edff3fd9-05f3-448f-8026-d635d186f009)

**Comment by cooperrc** [comment]
I need to add your netid to the server. I'll do that next class.
